apps/src/templates/teacherDashboard/SectionsPage.jsx

```diff
@@ -79,7 +83,8 @@ class SectionsPage extends Component {
   handleImport = courseId => {
     this.props.importClassroomStarted();
 
-    $.getJSON('/dashboardapi/import_google_classroom', { courseId }).then(() => {
+    const url = `/dashboardapi/import_${this.provider}_classroom`;
```

Comment:
My preference would be to have something like: `
const urlByProvider = {
  google: '/dashboardapi/import_google_classroom',
  clever: '/dashboardapi/import_clever_classroom'
};
const url = urlByProvider[this.provider];
```

This makes it so that I can search the endpoint and figure where it's being used.